examples/booktest/mysql/query.sql

```diff
@@ -46,3 +46,6 @@ UPDATE books
 SET title = ?, tags = ?, isbn = ?
 WHERE book_id = ?;
 
+/* name: DeleteAuthorBeforeYear :exec */
+DELETE FROM books
+WHERE yr < sqlc.arg(MinPublishYear) AND author_id = ? ;
```

Comment:
Can we change this to `min_publish_year` to make sure that the snake case to camel case conversion works?